airbyte-oauth/src/main/java/io/airbyte/oauth/flows/SmartsheetsOAuthFlow.java

```diff
@@ -23,14 +24,17 @@ public class SmartsheetsOAuthFlow extends BaseOAuth2Flow {
 
   private static final String AUTHORIZE_URL = "https://app.smartsheet.com/b/authorize";
   private static final String ACCESS_TOKEN_URL = "https://api.smartsheet.com/2.0/token";
+  private final Clock clock;
 
   public SmartsheetsOAuthFlow(final ConfigRepository configRepository, final HttpClient httpClient) {
     super(configRepository, httpClient);
+    this.clock = Clock.systemUTC();
   }
 
   @VisibleForTesting
   public SmartsheetsOAuthFlow(final ConfigRepository configRepository, final HttpClient httpClient, final Supplier<String> stateSupplier) {
```

Comment:
Note that oauth flows are being removed from this repo. These changes will have to be done on the airbyte-platform repo's airbyte-oauth module

Reply:
PR created: https://github.com/airbytehq/airbyte-platform/pull/41

Reply:
New PR in https://github.com/airbytehq/airbyte-platform-internal/pull/4813 👍🏻